search/search-client-connect/src/test/java/io/camunda/search/connect/os/OpensearchConnectorIT.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.connect.os;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.search.connect.configuration.ConnectConfiguration;
+import org.junit.jupiter.api.Test;
+
+public class OpensearchConnectorIT {
```

Comment:
Let's be more specific on naming.

```suggestion
public class AWSOpensearchConnectorIT {
```